Dockerfile-Admin

```diff
@@ -0,0 +1,14 @@
+FROM alpine:edge
```

Comment:
This seems to be an exact copy of the original dockerfile with just one line added. I am wondering if there is a more efficient way, e.g.:

- either put some kind of if/else-branching in the Dockerfile itself, see some ideas here: https://stackoverflow.com/questions/43654656/dockerfile-if-else-condition-with-external-arguments (also the second answer seems interesting, it links to this article, which seems worth a read: https://medium.com/@tonistiigi/advanced-multi-stage-build-patterns-6f741b852fae
- based on the previous ideas, maybe we could define all CoatRack images to be built in one dockerfile, so that we can also remove the "for loop over modules" in build-docker.sh?

Did not try any of that yet, this is just brainstorming, t.b.d.

Reply:
Done. The actual problem was, that RUN 'some if-command' would not have helped, because this works only within the container. I need a conditional COPY in the Dockerfile that copies the proxy.jar from the external host device into the coatrack-admin docker image. 

I think I found a elegant solution for this by saying: COPY the proxy.jar in every module, and then RUN an if-command which deletes it, if you are not the coatrack-admin module. Result: only coatrack-admin contains the proxy.jar and only one Dockerfile is required. Here are the crucial lines from the Dockerfile:

```
COPY spring-boot/proxy/target/coatrack-proxy-${MODULE_VERSION}.jar /opt/coatrack/lib/coatrack-proxy-${MODULE_VERSION}.jar
RUN if [ ${MODULE_NAME} != "coatrack-admin" ]; then rm /opt/coatrack/lib/coatrack-proxy-${MODULE_VERSION}.jar; fi
```